contributing-guides/style-guide.md

```diff
@@ -181,10 +181,15 @@ Use backticks on the following:
 
 ## Imperative Mood
 
-- Example descriptions have to be phrased in imperative mood.
+- All descriptions have to be phrased in the imperative mood.
 - For example, use `List all files` instead of `Listing all files` or `File listing`.
 - This also applies to all translations by default unless otherwise specified in the language-specific section below.
```

Comment:
```suggestion
**All descriptions have to be phrased in the imperative mood.** This also applies to all translations by default unless otherwise specified in the language-specific section below.
For example, when writing a documentation for `cd`, a tool to check out and work a specific directory in the Terminal or Command Prompt, **do not** write lengthy description such as:
> `cd` is a system tool, available in Windows, macOS, and Linux, to check out a specific directory to get things done in the Command Prompt, Terminam, and PowerShell.
It would be way easier for everyone to read:
> Change the current working directory.
If you are afraid of commands which may differ between platforms or operating systems (e.g. Windows vs macOS), most [tldr pages clients](https://github.com/tldr-pages/tldr/wiki/tldr-pages-clients) will choose the most suitable version of the command.
In this case, the information of the Windows version of `cd` (stored in `pages/windows/cd.md`) will be displayed by default to Windows users, and a generic/common version  (stored in `pages/common/cd.md`) for Linux, macOS, and others.
Similarly, when writing examples, instead of:
> > Listing all files:
> `ls`
Similarly, when writing descriptions for command examples, **check out for grammatical mistakes**. `Go to the specified directory` is preferred instead of:
* `Going to the specified directory` (should not be in present participle form)
* `This command will go to the specified directory` (it is clear that this example works for *this* comment
* `Let's go to the specified directory!`
* `Directory change` (use active form instead of passive, if possible)
```

Reply:
Great suggestion (I would love to make some minor fixes and tweaks), unfortunately GitHub's markdown rendering of codeblocks in suggestions makes it near impossible to edit them without conflicts.

I will mark this PR as draft for now, I will come back to this later tomorrow to make some modifications and apply this suggestion.